05-onchain.md

```diff
@@ -261,62 +300,63 @@ If the node receives (or already knows) a payment preimage for an
-when dealing with one's own commitment transaction.
+There are no delays constraining node behavior in this case, so it's simpler for
+a node to handle than the case in which it discovers its local commitment
+transaction.
 
 ## Requirements
 
-When a node sees a *commitment transaction* from the other node:
+When a node discovers a commitment transaction from the *remote node*:
 
 1. `to_remote`: No action is required; this is a simple P2WPKH output to us.
-   This output is considered *resolved* by the *commitment transaction* itself.
-2. `to_local`:: No action required; this is a payment to them.  This output is considered *resolved*
-   by the *commitment transaction*.
-3. HTLCs offered by this node: See "HTLC Output Handling: Their Commitment, Our Offers" below.
-4. HTLCs offered by the other node: See "HTLC Output Handling: Their Commitment, Their Offers" below.
-
-A node MUST handle the broadcast of any valid *commitment transaction*
-from the other node in this way; if it is unable to do so it MUST warn
+   This output is considered *resolved* by the commitment transaction itself.
+2. `to_local`:: No action required; this is a payment to them. This output is considered *resolved*
+   by the commitment transaction.
+3. HTLCs offered by the local node: See "HTLC Output Handling: Remote Commitment, Local Offers" below.
+4. HTLCs offered by the remote node: See "HTLC Output Handling: Remote Commitment, Remote Offers" below.
+
+A node MUST handle the broadcast of any *valid* commitment transaction
+from the *remote node* in this way; if it is unable to do so it MUST warn
 about lost funds.
 
 ## Rationale
 
-Note that there can be more than one valid, unrevoked *commitment
-transaction* after a signature has been received via `commitment_signed` and
+Note that there can be more than one valid, *unrevoked* commitment
+transaction after a signature has been received via `commitment_signed` and
 before the corresponding `revoke_and_ack`. Either commitment can serve as
-the other node's *commitment transaction*, hence the requirement to handle both.
+the *remote node's* commitment transaction, hence the requirement to handle both.
 
 In the case of data loss, a node can reach a state where it doesn't
-recognize all of the other node's commitment transaction HTLC outputs.  It can tell
+recognize all of the *remote node's* commitment transaction HTLC outputs. It can tell
 this has happened because the commitment number will be greater than
 expected, and because it has signed the transaction.
 If both nodes support `option-data-loss-protect` the node will
 know the peer's `per_commitment_point` and thus be able to derive its own
 `remotekey` for the transaction and salvage its own funds (but not the
 HTLCs).
 
-## HTLC Output Handling: Their Commitment, Our Offers
+## HTLC Output Handling: Remote Commitment, Local Offers
 
 Each HTLC output can only be spent by us, the offerer, after it's timed out,
```

Comment:
There's really no point in having this "us" now that you're changing to the local/remote dichotomy. I put it there solely to tie into the us/them in the headers.